### PR TITLE
	Add UT test for PolicyRuleBuilder

### PR DESCRIPTION
**Pull request by WanLinghao**
PolicyRuleBuilder is used to construct PolicyRule.
Add UT test to check if its construct is correct.	

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by jhorwit2** [comment]
Weird this ticket didn't get assigned to any of the owners. I wonder if it's because the approvers are empty...? Any who

/cc @liggitt

**Comment by WanLinghao** [comment]
@ericchiang  PTAL

File: pkg/apis/rbac/helpers_test.go [hunk 1]
```diff
@@ -175,3 +177,130 @@ func TestResourceMatches(t *testing.T) {
...
+	for _, tc := range tests {
+		actual, err := rbac.NewRule(tc.verbs...).Groups(tc.groups...).Resources(tc.resources...).Names(tc.names...).URLs(tc.urls...).Rule()
+		if err != nil {
+			if tc.expected {
+				t.Error(err)
+			} else {
+				continue
+			}
+		}
+		assert.Equal(t, tc.policyRule, actual, "%v: expected %v, got %v", tc.testName, tc.policyRule, actual)
```
Comment (liggitt) [inline pkg/apis/rbac/helpers_test.go]: prefer deepequal check, t.Error and continue on failure

**Pull request merged**